python_modules/libraries/dagster-pagerduty/dagster_pagerduty/resources.py

```diff
@@ -0,0 +1,25 @@
+import pypd
+
+from dagster import Dict, Field, ResourceDefinition, String
+
+
+class PagerDutyService:
+    def __init__(self, token):
+        self.token = token
+
+    def create_incident(self, summary):
```

Comment:
similar feedback to slack client. I think we should have a first class way of calling through to their underlying APIs and docstring should include point to url where they can see methods

Reply:
yeah, see my comment below, but this library doesn't do much - need to read PD API docs to understand what the supported API actually is. I updated this method name to match the underlying API call, though